tcc/src/main.c

```diff
@@ -107,33 +113,72 @@ main_start(char *prog)
 		{ "MD_OSVER",       OSVER          }
 	};
 
+	for (size_t i = 0; i < sizeof b / sizeof *b; i++) {
```

Comment:
We don't have C99 here yet; please keep the `size_t` variable at block scope. We'll need `make bootstrap-rebuild` to pass (it should fail on this I think)

Reply:
bootstrap-rebuild doesn't work for me in general, but was able to test this via CFLAGS=-std=c90, also brings up a couple of other things I probably need to fix